### PR TITLE
Add Travis CI build

### PR DESCRIPTION
**Pull request by KamataRyo**
I add CI service configuration and its badge on readme.
The build on it successed :)

### CODE REVIEW

**Comment by KamataRyo** [comment]
https://travis-ci.org/KamataRyo/wms/jobs/165431750

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (calvinmetcalf) [inline .gitignore]: you should add a global gitignore instead of per project like this

File: package.json [hunk 2]
```diff
@@ -22,6 +23,7 @@
     "url": "git@github.com:AppGeo/wms.git"
   },
   "devDependencies": {
+    "eslint": "^3.7.1",
```
Comment (calvinmetcalf) [inline package.json]: while we are at it we should probably fix the eslint because it doesn't currently work I don't think

**Comment by KamataRyo** [comment]
Thanks for taking time!
What is the difficulty with eslint? It looks to work well.
https://travis-ci.org/KamataRyo/wms/jobs/165431750#L343

Anyway, those lines were removed :)

**Comment by calvinmetcalf** [comment]
I think my .eslintrc was busted as it didn't actually load any rules, eslint stoped having default rules a bit ago, but that can be somethign for a diff pull, thank you for this

**Pull request merged**